main/src/main/java/org/mobilitydata/gtfsvalidator/runner/ValidationRunner.java

```diff
@@ -134,21 +134,22 @@ public Status run(ValidationRunnerConfig config) {
     }
     FeedMetadata feedMetadata = FeedMetadata.from(feedContainer, gtfsInput.getFilenames());
     closeGtfsInput(gtfsInput, noticeContainer);
+    feedMetadata.validationTimeSeconds = (System.nanoTime() - startNanos) / 1e9;
```

Comment:
Food for thought: I wonder if we could use [getProcessCpuTime](https://docs.oracle.com/en/java/javase/17/docs/api/jdk.management/com/sun/management/OperatingSystemMXBean.html#getProcessCpuTime()) to get amore accurate time estimate, since AFAIK it gives the actual time used by the process (ie without interference from other processes that ar charing the CPU).
Just an idea, I never used this myself.

Reply:
Created issue #1774 to explore your idea